esterni/piano-di-progetto/components/analisi-dei-rischi.tex

```diff
@@ -105,6 +105,12 @@ \subsection{Elenco dei rischi}%
   Monitoraggio:&
   \multicolumn{3}{p{38.5em}|}{Il proponente notificherà il gruppo in caso di problemi se possibile e in ogni caso la copia dell'applicazione sul server non sarà l'unica copia disponibile.}\\
   \hline % chktex 44
+  Problemi di comunicazione fra i membri di GruppOne \textbf{RPER004}& Essendo questo il primo progetto fortemente incentrato nella collaborazione per molti dei componenti del gruppo, si potranno presentare situazioni di incomprensione, dovute ad una poca od inefficace comunicazione fra i membri. &
```

Comment:
poca ed inefficace forse?

Reply:
Credo sia meglio "(..) dovute ad una comunicazione inefficiente ed inefficace (..)"
Visto che il proponente spinge molto su questi termini

Reply:
> poca ed inefficace forse?

Volevo evitare di dare un motivo preciso, per quello ho messo "o" invece che "e"

> Credo sia meglio "(..) dovute ad una comunicazione inefficiente ed inefficace (..)"
Visto che il proponente spinge molto su questi termini

Intanto è il committente, non il proponente. E poi inefficiente è un aggettivo particolare associato a comunicazione. Una comunicazione non efficiente è una comunicazione che si perde un sacco in chiacchiere, che non mi sembra sia il concetto giusto da passare in questo caso. Il problema non è (tanto) che ci perdiamo a parlare d'altro, quanto che quando parliamo uno dice una cosa, un altro ne capisce un'altra.